index.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const { initDB, initManagers } = require('./server/db/init_db.js');
+const routes = require('./server/routes.js');
+const bodyParser = require('body-parser');
+require('dotenv').config();
 const app = express();
-const port = 3000;
 
 initDB().then(() => {
 	const { postManager, userManager } = initManagers();
 	userManager.testAddUser();
 
-	app.use(express.static('client'));
-	app.listen(port, () => console.log('listening on %d', port));
+	app.use(bodyParser.urlencoded({ extended: true }));
+	routes(app, postManager, userManager);
```

Comment:
Routes should be called without passing parameters.

Reply:
Sorry, I meant that perhaps there is a way to import managers as singleton inside routes file. 
The app instance we need to pass anyway.